index.js

```diff
@@ -105,37 +90,49 @@ async function handleCommand(interaction) {
 		});
 	} else {
 		await interaction.deferReply();
-		await validate(false).then(async (value) => {
+		await validate(process.env.TWITCH_CLIENT_ID, process.env.TWITCH_CLIENT_SECRET, tokens.access_token, tokens.refresh_token, false).then(async (value) => {
```

Comment:
validate => validateTwitchToken.
Allows the reader to understand what the function does.